Makefile

```diff
@@ -16,69 +9,52 @@ all: docs packages
 tmp:
 	mkdir -p $(TMP)/.fmf
 
-
 # Run the test suite, optionally with coverage
 test: tmp
-	python3 -m pytest -vvv -ra --showlocals tests/unit
+	hatch run test:unit
 smoke: tmp
-	python3 -m pytest -vvv -ra --showlocals tests/unit/test_cli.py
-coverage: tmp
-	coverage run --source=tmt,bin -m pytest -vvv -ra --showlocals tests
-	coverage report
-	coverage annotate
+	hatch run test:smoke
+coverage: tmp nitrateconf
+	hatch run test:cov
+nitrateconf:
+	test -e ~/.nitrate || echo -en '[nitrate]\nurl = https://nitrate.server/xmlrpc/\n' | tee ~/.nitrate
+
 # Regenerate test data for integration tests
 # remove selected/all response files in tests/integration/test_data directory
 requre:
-	cd tests/integration; python3 -m pytest -vvv -ra --showlocals
-	# response files cleanup
-	requre-patch purge --replaces :milestone_url:str:SomeText --replaces :latency:float:0 tests/integration/test_data/test_nitrate/*
-
+	hatch run test:requre
 
 # Build documentation, prepare man page
 docs: man
```

Comment:
I don't think docs need man, right?

Reply:
I wouldn't say this represents a requirement, it's rather "man page is a documentation too, and should be built when running the top-level docs target". As in `docs: man html pdf ...`, use `docs` to build all documentation there is.

Reply:
why should it be built when building documentation?